doc/base/classes.xml

```diff
@@ -2091,12 +2115,14 @@
 			<argument index="0" name="axis" type="int">
 			</argument>
 			<description>
+				Returns the value of the given axis for things like triggers, touchpads, etc. that are embedded into the controller
```

Comment:
Many descriptions lack a dot at the end of the sentence.

Reply:
Yes punctuation isn't my strong point ;) let me have a look at those too